### ISSUE TITLE
Two hyporeflexia terms

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11402]
**HPO term**
https://hpo.jax.org/browse/term/HP:0001265 and https://hpo.jax.org/browse/term/HP:0001315 

**Reason for deprecation**
These seem to mean the same thing

**Suggested replacement or term to consider**
It seems these should be merged.

**Comment by pnrobinson** [comment]
[Reduced tendon reflexes (HP:0001315)](https://hpo.jax.org/browse/term/HP:0001315) and [Hyporeflexia (HP:0001265)
](https://hpo.jax.org/browse/term/HP:0001265).
@MickeySegal  -- the Reduced tendon reflexes was meant to be a grouping term for Hyporeflexia and Areflexia, i.e., "Absent or decreased deep tendon reflexes". Would there be a better way of naming the term or structuring this part of the ontology?

**Comment by MickeySegal** [comment]
Oh, I see.  Maybe a term such as deficient or subnormal would be better, or adding to the Comment section that this includes both hyporeflexia and areflexia.

**Comment by pnrobinson** [comment]
@MickeySegal thanks for pointing this out.
I have updated the term to (hopefully) clarify:

label: Diminished deep tendon reflex
def: A reduction (hyporeflexia) or complete absence (areflexia) of the involuntary muscle contraction normally elicited by a reflex stimulus, such as tapping a deep tendon.
